radix-engine-interface/src/api/object_api.rs

```diff
@@ -55,6 +55,14 @@ pub trait ClientObjectApi<E> {
         address: GlobalAddress,
     ) -> Result<(), E>;
 
+    fn globalize_with_address_and_child_object(
```

Comment:
I'd suggest we use `inner`/`outer` (and reserve `child`/`parent` for ownership).

So rename to `globalize_with_address_and_create_inner_object`?

Reply:
yup makes sense